docs/modules/ROOT/pages/meshing/module-3-centerlines-gui-tool.adoc

```diff
@@ -42,56 +57,77 @@ for example, using the provided Phantom example:
 
 `meshing_centerlinesmanagergui --input.surface.path Phantom/angiotk/surfacefromimage/model.stl --input.point-pair.path pointpair.data`
 
-IMPORTANT: when no point-pair file is specified, saving your progress will result is creation of two files: `<home>/feel/meshing_centerlinesmanagergui/np_1/<model-name>_pointpair.data`
+IMPORTANT: When no point-pair file is specified, saving your progress will
+result is creation of two files:
+`<home>/feel/meshing_centerlinesmanagergui/np_1/<model-name>_pointpair.data`
 `<home>/feel/meshing_centerlinesmanagergui/np_1/<model-name>_pointset.data`
 
 == Recommended workflow
 
-At this time (May 2016), center line generation is completely dependent and very sensitive to the provided pointsets.
-Therefore, a cautious stepwise workflow is strongly advised, and you should frequently check you progress with the center line generator and ParaView:
+At this time (May 2016), center line generation is completely dependent and very
+ sensitive to the provided pointsets.
+Therefore, a cautious stepwise workflow is strongly advised, and you should
+frequently check you progress with the center line generator and ParaView:
 
-- (optional) Prepare 2 different terminals, one for the CEGT, one for the center line generator.
+- (optional) Prepare 2 different terminals, one for the CEGT, one for the
+center line generator.
 
 === In the center line editing graphical tool
 - Start the GUI in one terminal
```

Comment:
I suggest that the keys appears first in the item or make a table with 3 columns: key, description and optional/required